__tests__/app.test.js

```diff
@@ -13,20 +13,16 @@ const endpointsData = require("../endpoints.json");
 beforeEach(() => seed({ topicData, userData, articleData, commentData }));
 afterAll(() => db.end());
 
-describe("/api/topics", () => {
-  test("GET:200 response array should contains data for all topics", () => {
+describe("/", () => {
+  test("should return a welcome message", () => {
     return request(app)
-      .get("/api/topics")
-      .expect(200)
-      .then(({ body: { topics } }) => {
-        expect(topics.length).toBe(3);
-        topics.forEach((topic) => {
-          expect(topic).toHaveProperty("description");
-          expect(topic).toHaveProperty("slug");
-        });
+      .get("/")
+      .then(({ body: { message } }) => {
+        expect(message).toBe("Welcome to news API!");
```

Comment:
Could possibly give them advice of where to go next? e.g. 'go to /api for endpoint info'